docs/resources/metric_condition.md

```diff
@@ -55,44 +56,41 @@ resource "lightstep_metric_condition" "beemo-requests" {
 
 ### Required
 
-- **expression** (Block List, Min: 1, Max: 1) (see [below for nested schema](#nestedblock--expression))
-- **metric_query** (Block List, Min: 1) (see [below for nested schema](#nestedblock--metric_query))
-- **name** (String)
-- **project_name** (String)
+- `expression` (Block List, Min: 1, Max: 1) (see [below for nested schema](#nestedblock--expression))
+- `metric_query` (Block List, Min: 1) (see [below for nested schema](#nestedblock--metric_query))
+- `name` (String)
+- `project_name` (String)
 
 ### Optional
 
-- **alerting_rule** (Block List) (see [below for nested schema](#nestedblock--alerting_rule))
-- **description** (String)
-- **id** (String) The ID of this resource.
+- `alerting_rule` (Block List) (see [below for nested schema](#nestedblock--alerting_rule))
+- `description` (String)
 
 ### Read-Only
 
-- **type** (String)
+- `id` (String) The ID of this resource.
+- `type` (String)
 
 <a id="nestedblock--expression"></a>
 ### Nested Schema for `expression`
 
 Required:
 
-- **evaluation_criteria** (String)
-- **evaluation_window** (String)
-- **operand** (String)
-- **thresholds** (Block List, Min: 1, Max: 1) (see [below for nested schema](#nestedblock--expression--thresholds))
+- `operand` (String)
+- `thresholds` (Block List, Min: 1, Max: 1) (see [below for nested schema](#nestedblock--expression--thresholds))
 
 Optional:
 
-- **is_multi** (Boolean)
-- **is_no_data** (Boolean)
-- **num_sec_per_point** (Number)
```

Comment:
what did this field do?

Reply:
Oops sorry. Meant to comment on this. I have no idea what this is, but the public API does not accept or return this field. Made sense to me to remove it while we were making breaking changes.